apps/web/public/locales/fi/navigation.json

```diff
@@ -13,5 +13,14 @@
   },
   "aria_logo": "Siirry kotisivulle",
   "aria_open_navigation": "Avaa navigaatio",
-  "aria_close_navigation": "Sulje navigaatio"
+  "aria_close_navigation": "Sulje navigaatio",
+  "settings": "Settings",
+  "analytics": "Tilastot",
+  "contracts": "Sopimukset",
+  "security": {
+    "title": "Turvallisuus",
```

Comment:
Should this instead be "Tietosuoja"? To me, "turvallisuus" sounds more like "physical safety" rather than "security".

Reply:
I think Tietosuoja fits better